.circleci/config.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     steps:
       - checkout
       - run: docker-compose up -d
+      - run: docker-compose exec api npm run migrate
```

Comment:
This shouldn't be necessary at present, but I don't see why we can't leave it in for later.